components/support/utils/src/main/java/mozilla/components/support/utils/ColorUtils.kt

```diff
@@ -23,6 +23,10 @@ object ColorUtils {
     @JvmStatic
     @SuppressWarnings("MagicNumber")
     fun isDark(@ColorInt color: Int): Boolean {
+        if (color == Color.WHITE || Color.alpha(color) < 128) {
+            return false
```

Comment:
So if the alpha is lower than 128 then we will use `Color.WHITE` in `getReadableTextColor()`. But that may depend on the background if that's a better choice or not?

Reply:
If the background colour is dark, we use a dark textColor? Shouldn't it be the opposite: if the background colour is dark, we use a light textColor? 🤔

Reply:
> > When isDark returns false, we use Color.Black as the text colour.
> 
> If the background colour is dark, we use a dark textColor? Shouldn't it be the opposite: if the background colour is dark, we use a light textColor? 🤔

Yes, if the background colour is dark (`isDark` is true) then we use `Color.WHITE`.

Reply:
Thanks, I think I recall now what this method does exactly. This looks good!